### PR TITLE
Feature/note block

### PR DESCRIPTION
**Pull request by akmiller01**

### CODE REVIEW

File: di_website/templates/blog/footnotes.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% load string_utils wagtailcore_tags %}
+{% if page.footnotes_list %}
+    <div class="m-notes">
+      <h3 class="m-notes__title">Notes</h3>
+      <ul class="m-notes__list">
+        {% for item in page.footnotes_list %}
+          <li><sup><a id="{{ item.footnote_id }}" href="#{{ item.source_id }}">{{ forloop.counter }}</a></sup>{{ item.text | urlize }}</li>
```
Comment (edwinmp) [inline di_website/templates/blog/footnotes.html]: I don't think it's supposed to have the `<sup>` element here; only in the text ... looks strange here

image

Comment (akmiller01) [reply di_website/templates/blog/footnotes.html]: Copied the markup directly from Surge @edwinmp 
image

http://development-initiatives.surge.sh/page-templates/12-blog-single

Comment (edwinmp) [reply di_website/templates/blog/footnotes.html]: Ah, in that case...

**Comment by akmiller01** [comment]
Cheers for the review @edwinmp but I'm DEFINITELY going to need to merge migrations here, if not wipe out the old ones and redo...

**Comment by akmiller01** [comment]
Ready for re-review @edwinmp

**Pull request merged**